tests/Http/GraphResponseTest.php

```diff
@@ -36,15 +35,6 @@ public function setUp(): void
         $this->response = new GraphResponse($this->request, "{response}", "200", ["foo" => "bar"]);
     }
 
-    public function testGetResponseAsObject()
-    {
-        $this->request->setReturnType(Model\User::class);
```

Comment:
We should still validate that setReturnType functions as expected. We should have some mock models that we use to test model related functionality.

When possible, and where meaningful, let's keep these model based tests and use a mock object so that we can validate the internals of the client.